bundles/org.openhab.ui/web/src/pages/settings/persistence/persistence-edit.vue

```diff
@@ -539,16 +674,32 @@ export default {
       this.checkDirty()
     },
     deleteModule (ev, module, index) {
+      if (!this.editable) return
       let swipeoutElement = ev.target
+      const self = this
```

Comment:
Normally there is no need for `self = this` "hacks" when using array functions in callbacks, which is the case here.
Why did you use this "hack" here?

BTW: `this.checkDirty` is used, so you anyway expect `this` to work fine.

Reply:
I changed it back. But it is strange. While it works without assigning to `self`, I cannot see the value of `this.persistence` on line 686 in the (Firefox) debugger. It is shown as undefined. I needed to see it when debugging.